.github/workflows/build.yml

```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
     types: [opened, synchronize, reopened]
   push:
-    branches: [master]
+    branches: [main]
```

Comment:
this is obviously correct for after we've switched to main from master.  However, I just wanted to confirm that it hasn't happened yet.  Thus do we want this in a different PR?

Reply:
Oops it's because I'm based from my test main branch. I will remove this later